docs/guides/react-server-cli.md

```diff
@@ -57,6 +57,35 @@ By default this is created at the site's root directory as `routes.json`.
 }
 ```
 
+You can also provide routes as a `routes.js` file.  The following routes.js
+file is equivalent to the json file above, but will work on Windows, in addition
+to OS X and Linux.
```

Comment:
Iirc, its not that `routes.json` doesn't work on Windows, its that windows uses / where everything else uses \ to delimit file paths fail, so while using a routes.json you have to provide Windows-compliant file paths or POSIX-compliant file paths, and that path.join smooths out those differences. It has been 3 or 4 months since we were dealing with the Windows failures, though, so it may have been something else that was horked -- I can pull the line if we wanna roll the dice.